build.sh

```diff
@@ -17,6 +17,11 @@ JULIA_LIB=$JULIA_H/../../lib
 # JLCXX_LIB must point to the directory that contains libcxxwrap_julia.so.0.x.x
 JLCXX_LIB=$JLCXX_H/../lib
 
+JULIA_H=C:/Users/dave/AppData/Local/Programs/Julia-1.7.0/include/julia
+JLCXX_H=C:/Users/dave/.julia/artifacts/c59cc3313b777d84d3bfeaa750b97bbd9f2b7d12/include
+JULIA_LIB=C:/Users/dave/AppData/Local/Programs/Julia-1.7.0/include/julia/../../lib
+JLCXX_LIB=C:/Users/dave/.julia/artifacts/c59cc3313b777d84d3bfeaa750b97bbd9f2b7d12/include/../lib
```

Comment:
Hard coded these paths because the Julia code above returns the paths with the wrong slashes \\ vs /